guessing-game.js

```diff
@@ -0,0 +1,26 @@
+var pry = require('pryjs');
+
+function GuessingGame(answer) {
+  this.answer = answer
+  this.guesses = [-1];
+  this.isSolved = function(){
+    for (var i = 0; i <= this.guesses.length; i++) {
```

Comment:
You mentioned this in class, but having to iterate through all the guesses each time can get pretty time consuming.